更新翻译v3.4天空盒 (#1958)
cocos-creator/creator-docs: en/concepts/scene/skybox.md

| Property | Description |
| :---| :--- |
| **Enabled** | Check this option to enable skybox. |
| **UseIBL** | Whether to use ambient lighting. Please refer to the **Using IBL** section below for more information. |
| **UseHDR** | If this option is checked, HDR (High Dynamic Range) will be turned on, if not checked, LDR (Low Dynamic Range) will be used. For details, please refer to the section **Switching HDR/LDR Mode** below. |
| **Envmap** | Environment map, TextureCube type, see below for details on how to set it. <br>When this property is empty, the skybox uses and displays pixel texture by default. |
| **ApplyDiffuseMap** | If this option is checked, the scene objects will use a more accurate diffuse map instead of the default hemispherical lighting. This option is only displayed when **UseIBL** is checked. Please refer to the **Diffuse Lighting** section below for details. |
| **DiffuseMap** | The convolution map for advanced diffuse lighting that is automatically generated when the **ApplyDiffuseMap** property is checked, and does not support manual editing. This option is only shown when **UseIBL** is checked. |

## Setting the Environment Map of the Skybox

> After opening the Skybox, it is also necessary to set the Skybox environment map. Drag and drop the map asset into the **Envmap** property box of the Skybox component, or click the arrow button behind the **Envmap** property box to select the desired map asset. If not set, the Skybox will use and display pixel maps by default.

Reviewer: ```suggestion
After enabling the Skybox, it is also necessary to set the environment map, which is used to generate ambient lighting in the scene. Drag and drop the map asset into the **Envmap** property box of the Skybox component, or click the arrow button behind the **Envmap** property box to select the desired map asset. If not set, the Skybox will use and display pixel maps by default.
```

The skybox supports the following environment map assets:

> 1. a single texture of type TextureCube, which can be set in Creator.

Reviewer: ```suggestion
1. A single texture of type TextureCube, which can be set in Creator.
```

> 1. a single texture of type TextureCube, which can be set in the Creator.

Reviewer 2: ```suggestion
1. a single texture of type TextureCube, which can be set in Creator.
```

    ![Set environment map for skybox](skybox/set-envmap.png)

> Then the setup is done. The developer can directly see the set environment map of the skybox in the **Scene** panel. If the map is not displayed correctly, you need to check if the value of **SkyIllum parameter** is too low, or **modify the Clear Flag** of Camera.

Reviewer 2: ```suggestion
The setup is done. The developer can directly see the set environment map of the skybox in the **Scene** panel. If the map is not displayed correctly, check if the value of **SkyIllum parameter** is too low, or **modify the Clear Flag** of Camera.
```

    ![Set CubeMap](skybox/cubemap-properties.png)

    > **Notes**:
    >
    > 1. The property boxes in CubeMap that do not have a texture asset yet will be populated using the default asset.
    > 2. The six property boxes in CubeMap **do not use the same texture**, otherwise they will not be displayed properly for some platforms.

## Using IBL

If the **UseIBL** property is checked, objects in the scene use specular reflections of the ambient lighting generated by the skybox, while allowing for more accurate diffuse lighting (convolution maps).

If the **UseIBL** property is disabled, objects in the scene will only use diffuse lighting in the hemispherical light mode.

### Diffuse Lighting

> The Creator supports the following two types of ambient diffuse lighting, which can be toggled by checking or unchecking the **ApplyDiffuseMap** property, noting that this property is only displayed when **UseIBL** is checked.

Reviewer 2: ```suggestion
Creator supports the following two types of ambient diffuse lighting, which can be toggled by checking or unchecking the **ApplyDiffuseMap** property, noting that this property is only displayed when **UseIBL** is checked.
```

    ![ambient-diffuse](skybox/ambient-diffuse.png)

2. **Convolution Map**: when the **ApplyDiffuseMap** property is checked, the convolution map diffuse will be used. This method is advanced diffuse reflection, which can correctly express the diffuse lighting generated by environment map, with better lighting directionality and details. However, the convolution map in the **DiffuseMap** property is automatically generated for diffuse reflection and does not allow manual editing.

    ![apply-diffuseMap](skybox/diffusemap-prop.png)

When using hemispherical light diffuse reflection, the effect is as follows:

![DiffuseMap](skybox/hemisphere-lighting.png)

When using diffuse reflection with convolution map, the effect is as follows. Compared with the above figure, it is obvious that the left backlighting surface is darker, highlighting the overall sense of hierarchy, and the contrast details between light and dark are also greatly improved.

![DiffuseMap](skybox/diffusemap.png)

> The contrast can be seen more clearly in the following GIF image:

Reviewer: ```suggestion
The contrast can be seen more clearly in the following GIF:
```

![Compare](skybox/compare.gif)

> **Notes**:
> 

> > 1. When replacing the environment map in the **Envmap** property, the Creator will automatically calculate the corresponding ambient lighting information, as well as the diffuse lighting (only CubeMap in the form of image files is supported, not including manually created CubeMap).

Reviewer: ```suggestion
> 1. When replacing the environment map in the **Envmap** property, Creator will automatically calculate the corresponding ambient lighting information, as well as the diffuse lighting (only CubeMap in the form of image files is supported, not including manually created CubeMap).
```

> 2. If the **UseIBL** property is checked, but **ApplyDiffuseMap** is not checked, the hemispheric light diffuse reflection will be used, and the effect is the same as if **UseIBL** is not checked.

## Toggling HDR/LDR mode

The **UseHDR** option in the Skybox component is used to toggle the HDR/LDR mode, which is used when checked.

![use-hdr](skybox/use-hdr.png)

> - HDR (High Dynamic Range): High dynamic range, with **photometric intensity of the light source** and **exposure parameters of the camera** to achieve a more realistic level of contrast between light and dark. If this mode is used, the intensity of all light sources (including parallel light, point light, spot light, etc.) **will become photometric physical units** and ambient light cube map should use **HDR format images** to provide a high dynamic range data source.

Reviewer: ```suggestion
- HDR (High Dynamic Range): High dynamic range, with **photometric intensity of the lights** and **exposure parameters of the camera** to achieve a more realistic level of contrast between light and dark. If this mode is used, the intensity of all lights (including parallel light, point light, spot light, etc.) **will become photometric physical units** and ambient light cube map should use **HDR format images** to provide a high dynamic range data source.
```

> - LDR (Low Dynamic Range): Low dynamic range. If this mode is used, the **light source intensity becomes unitless** and no longer has any connection to photometry or camera exposure. This mode is suitable for scenarios where you want the original map color to be reflected without any color tinting. Ambient light cube map can be done using images in formats such as **PNG**.

Reviewer: ```suggestion
- LDR (Low Dynamic Range): Low dynamic range. If this mode is used, the **lights intensity becomes unitless** and no longer has any connection to photometry or camera exposure. This mode is suitable for scenarios where you want the original map color to be reflected without any color tinting. Ambient light cube map can be done using images in formats such as **PNG**.
```
